docs/core_functions/yaml_preprocessor.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: ../_static/flatland_logo2.png
+    :width: 250px
+    :align: right
+    :target: ../_static/flatland_logo2.png
+
+Yaml Preprocessor
+==============================
+
+Flatland Server has a lua preprocessor for YAML with simple bindings for the environment variables and rosparam.
+The intent is to be able to build parametric models that are defined by dimensions and flags found in either environment variables or rosparam, in a similar way to xacro+urdf. Because this is parsed at model load time, any roslaunch rosparam loading will have completed and those parameters will be available.
+
+body/joint/plugin enabled flag
+------------------------------
+Model bodies, plugins and joints now have a new flag `enabled` which can be set to true or false either directly in the yaml, or based on more complex logic from a lua `$eval` string that returns "true" or "false". Disabled bodies, plugins and joints are skipped during yaml loading, and as a result are never instantiated. From Flatland's perspective `enabled: false` causes the affected body/plugin/joint to be deleted. 
+
+bindings for env and param
+-------------------------------
+
+Additional lua function bindings (beyond the normal standard libraries such as string, math, etc.):
```

Comment:
Are these bindings used for reading the environment variable and rosparam? If so can you explicitly say it?

Reply:
That's a good point! I'll add that in another pr.